api/application.go

```diff
@@ -246,7 +245,7 @@ func (h ApplicationHandler) Update(ctx *gin.Context) {
 	m.UpdateUser = h.BaseHandler.CurrentUser(ctx)
 	db = h.DB.Model(m)
 	db = db.Omit(clause.Associations)
-	db = db.Omit("Bucket")
+	db = db.Omit("BucketID")
```

Comment:
Bucket covered by _Associations_ and we never want to update the BucketID through the API.